slave.mk

```diff
@@ -84,6 +85,7 @@ export MULTIARCH_QEMU_ENVIRON
 export DOCKER_BASE_ARCH
 export CROSS_BUILD_ENVIRON
 export BLDENV
+export BUILD_WORKDIR
```

Comment:
Is there a reason for `BUILD_WORKDIR` to get exported?

Reply:
Yes, It is required to make MLNX_PATCH_LOC  an absolute path `MLNX_PATCH_LOC = $(BUILD_WORKDIR)/$(PLATFORM_PATH)/non-upstream-patches/patches.tar.gz`